Sources/HTTP/Serializer/ResponseSerializer.swift

```diff
@@ -0,0 +1,42 @@
+import Transport
+
+public final class ResponseSerializer<Stream: WriteableStream>: Serializer {
+    typealias StreamType = Stream
+    let stream: Stream
+    public init(_ stream: Stream) {
+        self.stream = stream
+    }
+    
+    public func serialize(_ response: Response) throws {
+        let message = try serialize(response as Message)
+        
+        let reasonPhrase = response.status.reasonPhrase.makeBytes()
+        let length = 5 // HTTP/
+            + 8 // 1.1_xxx_
```

Comment:
what is `_xxx_` here? I don't know what it corresponds to, and seems like it's variable, not 8, let's use a named constant describing what it is and what its doing.